bin/llvm-kompile-clang

```diff
@@ -256,7 +256,7 @@ if $link; then
       mkdir -p "$python_path"
 
       script_name="$(basename "$output_file").py"
-      cp "$LIBDIR/lldb/klldb.py" "$python_path/$script_name"
+      cp -f "$LIBDIR/lldb/klldb.py" "$python_path/$script_name"
```

Comment:
What did you need to make this change for?

Reply:
TL;DR - we're taking a shortcut in our use of Nix when running the integration tests and this is a workaround for that shortcut to work.

Longer version: `klldb.py` is installed by the Nix derivation that builds the LLVM backend; it has a path like `/nix/store/llvm-backend/.../lldb/klldb.py` and is _read-only_ at that path because it's an output of a build (if you could write to it, you could break Nix's guarantees about reproducibility and hashing).

When building a Nix derivation, you need to explicitly declare what its sources are; the Nix builder then takes care of setting permissions on those source inputs so that your build process can modify them inside a sandboxed build environment. However, when we copy this file we aren't copying it from our own sources, but rather straight out of the Nix store. It therefore retains its read-only permissions, and we need to force the copy to overwrite it.